src/Twig/SetcontentNode.php

```diff
@@ -29,7 +29,7 @@ public function compile(\Twig_Compiler $compiler)
 
         if (!is_null($this->getNode('wherearguments'))) {
             $compiler
-                ->raw(', $pager, ')
+                //->raw(', $pager, ')
                 ->subcompile($this->getNode('wherearguments'));
```

Comment:
If we can agree should we use or not that ``&$pager`` parameter then will remove... or not. :-)